src/test/java/org/codehaus/mojo/versions/utils/PropertiesVersionsFileReaderTest.java

```diff
@@ -0,0 +1,36 @@
+package org.codehaus.mojo.versions.utils;
+
+import junit.framework.TestCase;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Test;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.*;
+
+public class PropertiesVersionsFileReaderTest extends TestCase {
```

Comment:
```suggestion
public class PropertiesVersionsFileReaderTest {
```
`extends TestCase` is not needed when using JUnit 4